.build/docker/_docker_init_tests.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# inside the docker container, setup env before calling cassandra-test.sh
+export LANG=en_US.UTF-8
+export LC_CTYPE=en_US.UTF-8
+export PYTHONIOENCODING=utf-8
+export PYTHONUNBUFFERED=true
+export ANT_OPTS="${ANT_OPTS} -Dbuild.dir=${DIST_DIR}"
+
+pushd ${CASSANDRA_DIR} >/dev/null
```

Comment:
We don't check to ensure the various env var are set correctly here. Also - there's a lot of overlap in env var used between scripts; I think we should strongly consider moving those env var declaration to a central script that the others then source.